src/lib.rs

```diff
@@ -1031,6 +1031,19 @@ pub mod sync {
             self.0.get_unchecked()
         }
 
+        /// Get the mutable reference to the underlying value, without checking if the
+        /// cell is initialized.
+        ///
+        /// # Safety
+        ///
+        /// Caller must ensure that the cell is in initialized state, and that
+        /// the contents are acquired by (synchronized to) this thread.
+        #[inline]
+        #[cfg(not(feature = "critical_section"))]
+        pub unsafe fn get_mut_unchecked(&self) -> &mut T {
```

Comment:
What's the concrete use-case for this function? Given `& -> &mut` signature, this seems more or less impossible to use correctly.

Reply:
My initial purpose was to implement `wait_mut`, but I soon realized that `wait_mut` seemed meaningless. But yes, it should be `&mut -> &mut` here. Since there are `get_unchecked` and `get_mut`, will a `get_mut_unchecked` look reasonable?

Reply:
I think that’s already available as `get_mut().unwrap_unchecked()`

Reply:
Then what about a shortcut that's self-explanatory? I've added a new commit to fix the function signature, but if you believe this is not necessary, feel free to close this PR. Thanks.